src/Compilers/CSharp/Portable/Compilation/InitializerSemanticModel.cs

```diff
@@ -275,5 +275,20 @@ protected override void AnalyzeBoundNodeNullability(BoundNode boundRoot, Binder
         {
             NullableWalker.AnalyzeWithoutRewrite(Compilation, MemberSymbol, boundRoot, binder, diagnostics, createSnapshots);
         }
+
+        protected override bool IsNullableAnalysisEnabled()
+        {
+            switch (MemberSymbol.Kind)
+            {
+                case SymbolKind.Field:
+                case SymbolKind.Property:
```

Comment:
>case SymbolKind.Property [](start = 16, length = 24)

Can the symbol be a property? If I remember correctly, we always fall back to the backing field.

Reply:
The symbol can be a property. See line 55.

---
In reply to: [550246949](https://github.com/dotnet/roslyn/pull/49953#discussion_r550246949) [](ancestors = 550246949)

Reply:
> The symbol can be a property. See line 55.

I see. It looks like we are always using backing field for the Binder, but still use property for the model.

---
In reply to: [550276540](https://github.com/dotnet/roslyn/pull/49953#discussion_r550276540) [](ancestors = 550276540,550246949)